### PR TITLE
Added documentation for new selenium grid scaler

### PR DESCRIPTION
**Pull request by prashanth-volvocars**
Signed-off-by: prashanth <rprashanth27@gmail.com>

Added documentation for selenium grid scaler
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

File: content/docs/2.3/scalers/selenium-grid-scaler.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - type: selenium-grid
+    metadata:
+      url: 'http://selenium-hub:4444/graphql'
+      browserName: 'chrome'
+```
+
+**Parameter list:**
+
+- `url` is graphql url of your selenium grid. Refer to the selenium grid's documentation [here](https://www.selenium.dev/documentation/en/grid/grid_4/graphql_support/) to for more info.
+- `browserName` is the name of browser that usually gets passed in the browser capability.
```
Comment (zroubalik) [inline content/docs/2.3/scalers/selenium-grid-scaler.md:29]: Minor nit, is it possible to list the allowed values in here? Or add a link to the relevant selenium documentation>

File: content/docs/2.3/scalers/selenium-grid-scaler.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
++++
+title = "Selenium Grid Scaler"
+availability = "v2.3+"
```
Comment (zroubalik) [inline content/docs/2.3/scalers/selenium-grid-scaler.md:3]:
```suggestion
availability = "v2.4+"
```

File: content/docs/2.3/scalers/selenium-grid-scaler.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+description = "Scales selenium browser nodes based on number of requests waiting in session queue"
+layout = "scaler"
+go_file = "selenium_grid_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `selenium-grid` trigger that scales browser nodes based on number of requests in session queue.
+
+The scaler creates one browser node per pending request in session queue. You will have to create one trigger per browser capability that you would like to support in your selenium grid.
```
Comment (zroubalik) [inline content/docs/2.3/scalers/selenium-grid-scaler.md:14]:
```suggestion
The scaler creates one browser node per pending request in session queue. You will have to create one trigger per browser capability that you would like to support in your Selenium Grid.
```

File: content/docs/2.3/scalers/selenium-grid-scaler.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+triggers:
+  - type: selenium-grid
+    metadata:
+      url: 'http://selenium-hub:4444/graphql'
+      browserName: 'chrome'
+```
+
+**Parameter list:**
+
+- `url` is graphql url of your selenium grid. Refer to the selenium grid's documentation [here](https://www.selenium.dev/documentation/en/grid/grid_4/graphql_support/) to for more info.
```
Comment (zroubalik) [inline content/docs/2.3/scalers/selenium-grid-scaler.md:28]:
```suggestion
- `url` is graphql url of your Selenium Grid. Refer to the Selenium Grid's documentation [here](https://www.selenium.dev/documentation/en/grid/grid_4/graphql_support/) to for more info.
```

File: content/docs/2.3/scalers/selenium-grid-scaler.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  triggers:
+    - type: selenium-grid
+      metadata:
+        url: 'http://selenium-hub:4444/graphql'
+        browserName: 'chrome'
+```
+
+The above example will create chrome browser nodes equal to the requests pending in session queue for chrome browser.
+
+Similary for firefox
```
Comment (zroubalik) [inline content/docs/2.3/scalers/selenium-grid-scaler.md:56]:
```suggestion
Similary for Firefox
```

File: content/docs/2.3/scalers/selenium-grid-scaler.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  scaleTargetRef:
+    name: selenium-chrome-node
+  triggers:
+    - type: selenium-grid
+      metadata:
+        url: 'http://selenium-hub:4444/graphql'
+        browserName: 'chrome'
+```
+
+The above example will create chrome browser nodes equal to the requests pending in session queue for chrome browser.
```
Comment (zroubalik) [inline content/docs/2.3/scalers/selenium-grid-scaler.md:54]:
```suggestion
The above example will create Chrome browser nodes equal to the requests pending in session queue for Chrome browser.
```

**Review by zroubalik — CHANGES_REQUESTED** [review]
Looking good, minor nits though.

Could you move the documentation to `content/docs/2.4/scalers/selenium-grid-scaler.md`?
2.3 has already been released.

File: content/docs/2.3/scalers/selenium-grid-scaler.md (hunk 1 above)
Comment (prashanth-volvocars) [reply content/docs/2.3/scalers/selenium-grid-scaler.md]: Added link to the documentation

File: content/docs/2.3/scalers/selenium-grid-scaler.md (hunk 2 above)
Comment (prashanth-volvocars) [reply content/docs/2.3/scalers/selenium-grid-scaler.md]: Changed the version to 2.4

File: content/docs/2.3/scalers/selenium-grid-scaler.md (hunk 3 above)
Comment (prashanth-volvocars) [reply content/docs/2.3/scalers/selenium-grid-scaler.md]: done

File: content/docs/2.3/scalers/selenium-grid-scaler.md (hunk 4 above)
Comment (prashanth-volvocars) [reply content/docs/2.3/scalers/selenium-grid-scaler.md]: done

File: content/docs/2.3/scalers/selenium-grid-scaler.md (hunk 5 above)
Comment (prashanth-volvocars) [reply content/docs/2.3/scalers/selenium-grid-scaler.md]: done

File: content/docs/2.3/scalers/selenium-grid-scaler.md (hunk 6 above)
Comment (prashanth-volvocars) [reply content/docs/2.3/scalers/selenium-grid-scaler.md]: done

File: content/docs/2.4/scalers/selenium-grid-scaler.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
++++
+title = "Selenium Grid Scaler"
+availability = "v2.4+"
+maintainer = "Volvo Cars"
+description = "Scales selenium browser nodes based on number of requests waiting in session queue"
```
Comment (zroubalik) [inline content/docs/2.4/scalers/selenium-grid-scaler.md:5]:
```suggestion
description = "Scales Selenium browser nodes based on number of requests waiting in session queue"
```

**Review by zroubalik — COMMENTED** [review]
Looking good! Sorry I missed this very minor nit on the previous review 🤦‍♂️

**Review by zroubalik — APPROVED** [review]
LGTM, thanks for the nice docs.

**Pull request merged**